src/main/resources/db/changelog/changeset/create-default-admin-and-user.sql

```diff
@@ -1,6 +1,6 @@
 -- liquibase formatted sql
--- changeset Millrocious:create-default-admin-and-user
-INSERT INTO "user" (name, email, password, role, locale_id, subscription, update_date, create_date) VALUES
```

Comment:
If you will change migrations like that, the application won't start because Liquibase will fail on the checksums validation step

Reply:
So, I have to insert additionally `-- changeset MrZhenShen` above the `-- changeset Millrocious`, bot rather just change?